book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(_title, author)
+    @title = label
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] As required, let's ensure that a method for adding rental is present. The _book_ itself should be sent as a parameter to create the new _rental_, thereby achieving the _'has-many'_ association. We can achieve this by modifying our code to look like the following example below:

```ruby
class Book
  attr_accessor :title, :author, :rentals
  .
  .
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```